fv3core/stencils/d_sw.py

```diff
@@ -55,6 +54,24 @@ def flux_adjust(
         w = flux_integral(w, delp, gx, gy, rarea)
 
 
+@gtstencil()
+def flux_capacitor(
+    cx: FloatField,
+    cy: FloatField,
+    xflux: FloatField,
+    yflux: FloatField,
+    crx_adv: FloatField,
+    cry_adv: FloatField,
+    fx: FloatField,
+    fy: FloatField,
+):
```

Comment:
I'm curious what this does. Please fill the docstring out as much as you can and add a header description. It's OK if some/most are still `???`.

I would never have guessed with a name of "capacitor" that all this does is add its inputs to other inputs.

```suggestion
):
    """
    Args:
        cx: ??? (inout)
        cy: ??? (inout)
        xflux: ??? (inout)
        yflux: ??? (inout)
        crx_adv: ??? (in)
        cry_adv: ??? (in)
        fx: ??? (in)
        fy: ??? (in)
    """
```

Reply:
This was just a direct move of existing code. It's the final step computing the flux capacitor fields, even though the math is simple. sure I can update the docstring, doing that always extend the time to work on a task quite a lot.